apiserver/facades/client/machinemanager/machinemanager.go

```diff
@@ -177,6 +186,7 @@ func NewMachineManagerAPI(
 	backend Backend,
 	cloudService common.CloudService,
 	credentialService common.CredentialService,
+	machineService MachineService,
```

Comment:
This is becoming silly. Can we change the constructor (can be another PR) that takes a config.

Reply:
yes, we do need to start doing that moving forward. it is getting out of hand